### ISSUE TITLE
Feature Request: requestImageForAsset

### ISSUE THREAD

**Issue by BlenderDude** [olofd/react-native-photos-framework#72]
Add the requestImageForAsset function of the UIImage component to allow your plugin to access the jpg version of heic images. HEIC is really making developers cry.

**Comment by aforty** [comment]
This would be helpful!

**Comment by srameshr** [comment]
Yes. This will make this plugin usable for most non ios developers. I have no idea what to make of ```photos://``` urls. ```Image editor``` plugin that I use don't accept ```photos://``` url. Can't display ```Gifs``` with this. Could you consider this as a priority and add the same.

**Comment by srameshr** [comment]
Any updates on this?

**Comment by BlenderDude** [comment]
I made a workaround for my personal app that I made. Takes the image asset and converts into a JPEG regardless of the original file format. I'll make a PR sometime this weekend to help out you guys.

**Comment by srameshr** [comment]
Cool!
Please do share it.
